pydicom/jsonrep.py

```diff
@@ -97,15 +97,20 @@ def __init__(
         self.vr = vr
         self.value = value
         self.value_key = value_key
-        if (
-            bulk_data_uri_handler and
-            len(signature(bulk_data_uri_handler).parameters) == 1
-        ):
-            def wrapped_bulk_data_handler(tag, vr, value):
-                return bulk_data_uri_handler(value)
-            self.bulk_data_element_handler = wrapped_bulk_data_handler
+        self.bulk_data_element_handler: Callable[[BaseTag, str, str], Any]
```

Comment:
Typing `jsonrep` was really not working, so I gave up and just fixed the errors.